CHANGELOG

```diff
@@ -1,3 +1,7 @@
+* Fri Nov 03 2017 Trevor Vaughan <tvaughan@onyxpoint.com> - 1.0.4-0
+- Fix parameter issues in simp_options::ldap
+- Fix PE support in simp_options::puppet
```

Comment:
It was not a fix

Reply:
@jeefberkey Yes, it was. That option should have always flipped based on the PE facts since they are available. It was an oversight that it got into place as it was.